plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/options/LayerUnzippingStrategy.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 
+ *******************************************************************************/
+package org.eclipse.elk.alg.layered.options;
+
+/**
+ * Strategies for unzipping layers i.e. splitting up nodes into multiple layers to
+ * create more compact drawings.
+ *
+ */
+public enum LayerUnzippingStrategy {
+    
+    NONE,
+    /** Splits all layers with more than two nodes into two layers. */
+    N_LAYERS;
+
+}
```

Comment:
Usually this is a factory. Why did you decide differently for layer unzipping?

Reply:
I couldn't find any other intermediate processors, which actually do this in their own strategy. They all seem to use simple enums such as this and then the factory is implemented by `IntermediateProcessorStrategy`. I can do it differently but that would break the current norm.